app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -4,9 +4,37 @@
 
 package org.mozilla.fenix.perf
 
+import androidx.core.view.doOnPreDraw
+import kotlinx.android.synthetic.main.activity_home.*
+import org.mozilla.fenix.HomeActivity
+
 /**
  * A collection of objects related to app performance.
  */
 object Performance {
     const val TAG = "FenixPerf"
+
+    /**
+     * Instruments cold startup time for use with our internal measuring system, FNPRMS. This may
+     * also appear in Google Play Vitals dashboards.
+     *
+     * This will need to be rewritten if any parts of the UI are changed to be displayed
+     * asynchronously.
+     *
+     * In the current implementation, we only intend to instrument cold startup to the homescreen.
+     * To save implementation time, we ignore the fact that the RecyclerView draws twice if the user
+     * has tabs, collections, etc. open: the "No tabs" placeholder and a tab list. This
+     * instrumentation will only capture the "No tabs" draw.
+     */
+    fun instrumentColdStartupToHomescreenTime(activity: HomeActivity) {
+        // For greater accuracy, we could add an onDrawListener instead of a preDrawListener but:
+        // - single use onDrawListeners are not built-in and it's non-trivial to write one
+        // - the difference is timing is minimal (< 7ms on Pixel 2)
```

Comment:
uber nit: typo -- in not is.

Reply:
I'll address this in my next PR so we can just land this one without re-running CI.

Reply:
There was no second PR so I made it: https://github.com/mozilla-mobile/fenix/pull/7848